.github/workflows/check-pr-title.yml

```diff
@@ -11,38 +11,30 @@ jobs:
     steps:
       # check the basic semantic compliance of the PR title
       # https://github.com/amannn/action-semantic-pull-request/releases
-      - uses: amannn/action-semantic-pull-request@v5.5.2
+      - uses: amannn/action-semantic-pull-request@v5.5.3
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          # Allowed types: add a notion doc
           types: |
             feat
             fix
-            docs
```

Comment:
let's keep

Reply:
i thought about combining it with style but open to keeping it... trying to reduce the number of types so it is easier to remember and/or map